controllers/hetznerbaremetalhost_controller_test.go

```diff
@@ -243,7 +243,7 @@ var _ = Describe("HetznerBareMetalHostReconciler", func() {
 			By("making sure the it has been deleted")
 			Eventually(func() bool {
 				return apierrors.IsNotFound(testEnv.Get(ctx, key, host))
-			}, timeout, time.Second).Should(BeTrue())
+			}, 10*time.Second, time.Second).Should(BeTrue())
```

Comment:
We have talked about this already. We should find out the true reason why this is flaky. 10 seconds won't help

Reply:
@janiskemper I agree. I could not reproduce it. On my machine, the test was successful in 1s. I will use the old timeout again.